190114_tema/src/ro/sci/teo/ian14/Main.java

```diff
@@ -0,0 +1,110 @@
+package ro.sci.teo.ian14;
+
+import java.util.*;
+
+/**
+ * This is the main class, where all the program parts are put together.
+ *
+ * @author Teo
+ */
+public class Main {
+    public static void main(String[] args) {
+        //QUESTIONS:
+        //How do i make the product´s id unique? I considered it unique by default.
+        //How would you use an interface here?
```

Comment:
here? for the main? I wouldn't.

The interface for the Store is a good idea. The methods aren't perfect... it adds items from files but does not ask for the file location it just has one hardcoded
it adds items from the console without specifying in the method name or javadoc.
I would just have the store manage the lists of stuff:
List<Item>
List<Order> 
and expose methods for:
void addProduct(Product, quantity)
boolean sellProduct(Product, quantity) // returns false if invalid operation
String getItemReport()  // returns a formatted string with all the products and their quantities
String getOrderReport(LocalDate date)  // returns a formatted string with all the orders from a day